composer.json

```diff
@@ -21,13 +22,15 @@
         "spryker/refund": "^5.0.0",
         "spryker/router": "^1.6.0",
         "spryker/sales": "^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^10.0.0 || ^11.0.0",
+        "spryker/shipment": "^8.0.0",
```

Comment:
Yes, It looks like major change.
But in this case we already have this dependency via `Shop.CheckoutPage -> [spryker/shipment] => ^7.0.0 || ^8.0.0`.
So, to avoid major we should support also `^7.0.0` version.
Please check if the last minor version of `7.0.0` version contains the methods and data structures used in the current PR.
Then fix this dependency to `"spryker/shipment": "^7.0.0 ||^8.0.0",`